sentences/en/lock_HassTurnOff.yaml

```diff
@@ -3,14 +3,14 @@ intents:
   HassTurnOff:
     data:
       - sentences:
-          - "unlock <name> [in <area>]"
+          - "unlock [the] <name> [in <area>]"
         requires_context:
           domain: lock
         response: lock
 
       - sentences:
-          - "unlock [all] [the] [(lock|door)[s]] [in] <area>"
-          - "unlock [all] <area> [(lock|door)[s]]"
+          - "unlock [all] [the] [<lockable>] [in] <area>"
+          - "unlock [all] <area> [<lockable>]"
```

Comment:
If all optional parts are missing, these 2 sentence templates are the same: `unlock <area>`. Please avoid this.